bootstrap/004_labels.sql

```diff
@@ -263,114 +166,66 @@ END;
 
 CREATE OR REPLACE PROCEDURE ADMIN.DELETE_LABEL(name text)
     RETURNS text
-    LANGUAGE SQL
+    language python
+    runtime_version = "3.10"
+    handler = 'delete_label'
+    packages = ('snowflake-snowpark-python', 'pydantic')
+    imports = ('{{stage}}/python/crud.zip')
     EXECUTE AS OWNER
 AS
-BEGIN
-    if (:name is null) then
-      return 'Name must not be null.';
-    end if;
-
-    DELETE FROM internal.labels where name = :name;
-    CALL INTERNAL.UPDATE_LABEL_VIEW();
-    return 'done';
-END;
+$$
+from crud import delete_entity
+def delete_label(session, name):
+    return delete_entity(session, 'LABEL', name)
+$$;
 
+-- TODO Differentiate from ungrouped labels  by having "AND IS_DYNAMIC" in the delete clause
 CREATE OR REPLACE PROCEDURE ADMIN.DELETE_DYNAMIC_LABEL(name text)
     RETURNS text
-    LANGUAGE SQL
+    language python
```

Comment:
why have this? Its the same is it not?

Reply:
I had consolidated the `DELETE_DYNAMIC_LABEL` and `DELETE_LABEL` last weekend, and then regretted doing it in the first pass. I left them separate for this PR, but completely agree that we don't need two long-term.

Reply:
foolow on is perfect